Automattic/jetpack
# Boost: Add "load default strings" to excludes in concatenate UI

## Proposed changes:

* Add a button, similar to cornerstone pages, that loads default exclude values for concatenate JS and CSS;
* Clean up CSS in minify component.

![CleanShot 2024-12-06 at 16 59 15](https://github.com/user-attachments/assets/0cdc6f94-951b-4281-8736-0bb99e2eaa50)


### Other information:


## Jetpack product discussion

n/a

## Does this pull request change what data or activity we track or use?

no

## Testing instructions:

- Setup Boost;
- Make sure concatenate JS/CSS UI is behaving as expected.
